src/components/sidebar/routePathView/routePathListTab/RoutePathListLink.tsx

```diff
@@ -52,6 +55,79 @@ class RoutePathListLink extends React.Component<IRoutePathListLinkProps> {
                         value={rpLink.isStartNodeTimeAlignmentStop ? 'Kyllä' : 'ei'}
                     />
                 </div>
+                <div className={s.flexRow}>
+                    <div className={s.inputLabel}>
+                        ALKUSOLMUN SARAKE NRO
+                    </div>
+                </div>
+                <div className={s.flexRow}>
+                    <div className={s.flexInnerRow}>
+                        <input
+                            placeholder='1'
+                            type='text'
+                            className={s.smallInput}
+                        />
+                        <Checkbox
+                            checked={false}
+                            text={'Ohitusaika kirja-aikat.'}
+                            onClick={this.onChange}
+                        />
+                    </div>
+                    <div className={s.flexInnerRow}>
+                        <input
+                            placeholder='1'
+                            type='text'
+                            className={s.smallInput}
+                        />
+                        <Checkbox
+                            checked={false}
+                            text={'Ohitusaika nettiaikat.'}
+                            onClick={this.onChange}
+                        />
+                    </div>
+                </div>
+                <div className={s.flexRow}>
+                    <div className={s.inputLabel}>
+                        VIIM. LINKIN LOPPUSOLMU SARAKE NRO
+                    </div>
+                </div>
+                <div className={s.flexRow}>
+                    <div className={s.flexInnerRow}>
+                        <input
+                            placeholder='1'
+                            type='text'
+                            className={s.smallInput}
+                        />
+                        <Checkbox
+                            checked={false}
+                            text={'Ohitusaika nettiaikat.'}
+                            onClick={this.onChange}
+                        />
+                    </div>
+                    <div className={s.flexInnerRow}>
+                        <input
+                            placeholder='1'
+                            type='text'
+                            className={s.smallInput}
+                        />
+                        <Checkbox
+                            checked={false}
+                            text={'Ohitusaika kirja-aikat.'}
+                            onClick={this.onChange}
+                        />
+                    </div>
+
+                </div>
+                <div className={s.flexRow}>
+                    <Dropdown
+                        label='SOLMU HASTUS-PAIKKANA'
+                        items={['Kyllä', 'Ei']}
```

Comment:
Hmm, need to check this but should this be a key: value array? Maybe a "yesNoCodeList" could be used here.

Reply:
The problem with yesNoCodeList, is that it reflects a boolean, and you can not have booleans as keys in a codelist. We could add a new component for boolean values, that we could use in other places as well.

Reply:
https://github.com/HSLdevcom/jore-map-ui/issues/624